docs/UserGuide.md

```diff
@@ -2,1012 +2,1171 @@
-<br><i>Note how the app contains some sample data.</i>
-5. You will start on the `Course page` (Refer to [<i>Figure 3</i>](#navigation-guide) below for the Navigation Guide).
-6. Type your command in the command box (Refer to [<i>Figure 2</i>](#visual-guide) below for the Visual Guide) and press Enter/Return to execute it.
-<br>e.g. typing `help` and pressing the Enter/Return key will open the help window.
-    <br>Some example commands you can try:
+---
+
+## Getting started
+
+### Setting up
+#### Installation
+1. Ensure you have Java 11 or above installed on your Computer.
+2. Download the latest `clipboard.jar` from [here](https://github.com/AY2223S2-CS2103T-T15-4/tp/releases).
+3. Copy `clipboard.jar` to the folder you want to use as the home folder of CLIpboard.
+
+#### Launching CLIpboard
+*Option 1*:
+* Double-click on `clipboard.jar` to run it. If this does not work, go to *Option 2*.
+
+*Option 2*:
+1. Open a command terminal. [Guide to open a command terminal](https://www.ionos.com/help/email/troubleshooting-mail-basicmail-business/access-the-command-prompt-or-terminal/#:~:text=Open%20Command%20Prompt%20in%20Windows,cmd%22%20and%20then%20click%20OK.).
+2. Copy the file path of the **folder** you put `clipboard.jar` in.
+    * [Guide For Windows](https://www.howtogeek.com/670447/how-to-copy-the-full-path-of-a-file-on-windows-10/)
+    * [Guide For MacOS](https://support.apple.com/en-sg/guide/mac-help/mchlp1774/mac#:~:text=Show%20the%20path%20to%20a%20file%20or%20folder&text=Choose%20View%20%3E%20Show%20Path%20Bar,bottom%20of%20the%20Finder%20window.)
+    * [Guide For Linux](https://www.tutorialspoint.com/get-the-full-path-of-a-file-in-linux)
+3. In the command terminal, type `cd <FILE_PATH>`, where `<FILE_PATH>` is the file path you found in step 2.
+4. Type `java -jar clipboard.jar` into the command terminal and press `Enter` to launch CLIpboard.&nbsp;
+
+If you see something like this, you're good to go!<br>
+<img src="./images/Ui.png" width="650" height="529"/>
+<br>
+*CLIpboard contains some sample data if there is no initial data, as is the case when you first launch it.*
+
+### Quick Start
+
+1. When you first launch CLIpboard, you will be brought to the `Course page`.
+2. Type your command in the `command box` and press `Enter/Return` to execute it.
+   <br>e.g. typing `help` and pressing the `Enter/Return` key will open the help window.
+   <br>Some example commands you can try:
    1. `select 1` : Selects the first course displayed (eg. `CS2103T`) on the `Course Page`. You will be redirected to the `Group Page`.
    2. `select 1` : Selects the first group displayed (eg. `T15`) on the `Group Page`. You will be redirected to the `Student Page`.
    3. `add student n/John Doe p/98765432 e/johnd@example.com sid/A1234567X` : Adds a student named `John Doe` with the particulars into the list on the `Student Page`.
    4. `delete student 3` : Deletes the 3rd student (eg. `Lim Kim Choo`) shown in the student list.
```

Comment:
```suggestion
   4. `delete student 3` : Deletes the *third* student (eg. `Lim Kim Choo`) shown in the student list.
```

For consistency with updated document.

Reply:
fixed, and replaced all other instances of numbered sequence to words